c2cgeoform/__init__.py

```diff
@@ -51,7 +53,13 @@ def includeme(config):
     This function creates routes and views for c2cgeoform pages.
     """
     config.include('pyramid_chameleon')
-    config.include('pyramid_beaker')  # use Beaker for session storage
+    try:
+        config.include('pyramid_beaker')  # use Beaker for session storage
+    except ModuleNotFoundError:
+        logger.info("""Unable to include pyramid_beaker this probably mean that he is not installer.
```

Comment:
Shouldn't we log this as warning ?

```suggestion
        logger.info("""Unable to include pyramid_beaker this probably mean that it is not installed.
```